frontend/packages/dev-console/integration-tests/features/vulnerability/project_vulnerability_ODC-4882

```diff
@@ -0,0 +1,93 @@
+Feature: Image Vulnerability in Project
+    As a user, I would like to see Image Vulnerability in Project
+
+Background:
+    Given user is in Developer perspecitve
+    And user in project namespace "aut-image-vulnerabilty"
+    And quay container security operator installed
+
+@regression
+Scenario: Vulnerability in project overview    
+   Given user is in topology page
+   When user goes to project tab on navigation menu
+   Then user can see vulnerabilities in Status section
+   And vulnerabilities has number associated with it
+
+@regression
+Scenario: Image Vulnerability breakdown pop over in project overview    
+   Given user is in project tab
+   When user clicks on the Image Vulnerabilities
+   Then user can see Image Vulnerability breakdown pop over
+   And user can see Vulnerable Container Images and Fixable Container Images
+   And user can see View all link on bottom of pop over
+
+@regression
+Scenario: Visit Image Manifest Vulnerabilities page from status in project overview    
+   Given user is in project tab
+   When user clicks on the Image Vulnerabilities in Status section
+   And user clicks on the View all link at the bottom of Image Vulnerability breakdown pop over
+   Then user can see Image Manifest Vulnerabilities page
+
+@regression
+Scenario: Visit ImageManifestVuln from status in project overview    
+   Given user is in project tab
+   When user clicks on the Image Vulnerabilities in Status section
+   And user clicks on the image name in Image Vulnerability breakdown pop over to open that particular ImageManifestVuln
+   Then user can see ImageManifestVuln for the image name
+
+@regression
+Scenario: Visit quay page from status in project overview    
+   Given user is in project tab
+   When user clicks on the Image Vulnerabilities in Status section
+   And user clicks on the fixable link in Image Vulnerability breakdown pop over
+   Then user can see the image open in quay.io
+
+@regression
+Scenario: Vulnerability tab in developer console    
+   Given user is in topology page
+   When user goes to project tab on navigation menu
+   Then user can see vulnerability tab
+   And user clicks on the vulnerability tab
```

Comment:
This is the action and it should not be in Then statement

Reply:
This is a required action

Reply:
I think @sanketpathak - updated it